sqlitedict.py

```diff
@@ -98,7 +98,13 @@ def encode(obj):
 
 def decode(obj):
     """Deserialize objects retrieved from SQLite."""
-    return loads(bytes(obj))
+    try:
+        return loads(bytes(obj))
+    except (UnpicklingError, ValueError, EOFError, TypeError):  # Checking for the possible DB used by version 1.2.
+        logger.warning("Not a pickled string %s in sqlite, probably compatibility with sqlite 2.0 problem. "
+            "you can simply run `python sqlite_migration.py` ... in order to migrate the sqlite DB", obj)
+        raise KeyError("your sqlite db is not compatibile with sqlitedict2.0 "
+            "you can simply run `python sqlite_migration.py` ... in order to migrate the sqlite DB")
```

Comment:
"Simply run `python sqlite_migration.py`" .. sure, if you happen to be using the sqlitedict project directly from github.  It is not distributed with the sqlitedict package.

I read this full phrase several times, even wrote a test case for it, and some 30 minutes later I only **now** understand the meaning of the ellipses ``...`` is that extra unspecified arguments would be required.

Woah boy! And english is my first language, I can't imagine the difficulty this might have for others. I propose instead a URL link after the anchor tag of matching header for "Migrating sqlitedict data"  or some such header at github's README.rst file.

I don't see any such documentation, this is absolutely required. Please consider a python novice who was not even aware of sqlitedict of software he is using just got this error:

- using filesystem locate tools for sqlite_migration.py, fails to find it
- uses google, god knows what they'll find, maybe projects like https://github.com/ottowayne/sqlite-migration-fix
- the log string does not mention sqlitedict, but luckily the exception does, so they google that
- First result https://pypi.python.org/pypi/sqlitedict/1.2.0 makes no mention of sqlite_migration.py
- out of frustration, finally finds "Comments, bug reports" to say "My damn program stopped working!", and clicks the github url.
- in the file browser, sees sqlite_migration.py. Maybe file a complaint anyway, this took a while to find!